### ISSUE TITLE
[Accessibility][PDF Viewer] The Adobe Acrobat link in the controls dock is not focusable or keyboard operable.

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1743]
**Steps to Reproduce**
1. Open page in Google Chrome.
2. Press the Tab key to attempt to navigate to the Adobe Acrobat icon link using the keyboard.

**Current Behavior**
The Adobe links are excluded from the focus order. These controls can be clicked on with a mouse to open the Adobe Acrobat Home page in a new tab, but are marked up as <div> elements and cannot receive focus.

**Expected behavior/code**
All interactive elements can receive keyboard focus and can be activated using a keyboard alone.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Keyboard users

**WCAG SC**
2.1.1 Keyboard

**Code snippet**
```
<div class="HUDView__externalComponentLeftUndocked___3ia4i">
    <div class="sdk-Branding-branding sdk-Branding-hudBranding" role="presentation">
        <svg data-name="Layer 1" viewBox="0 0 24 24" focusable="false" aria-hidden="true" role="img" class="spectrum-Icon spectrum-Icon--sizeM">...</svg> 
    </div>
</div>
```

**Possible Solution**
If possible, use a native HTML anchor element with a valid href value, which will provide role information and keyboard accessibility by default. If this is not possible, ensure that custom controls can receive focus by adding tabindex="0", provide appropriate role information (in this case, role="link"), and can be activated using expected keyboard interactions (links should be activated with the Enter key).

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by glasert** [comment]
This issue won't be fixed as the acrobat team is working on a modern viewer which addresses accessibility and will replace the current viewer in 2022. Therefore we defer and close this issue
